WordPress/Classes/Models/BloggingPrompt+CoreDataClass.swift

```diff
@@ -21,17 +21,17 @@ public class BloggingPrompt: NSManagedObject {
         BloggingPromptsAttribution(rawValue: attribution.lowercased())
     }
 
-    /// Convenience method to map properties from `RemoteBloggingPrompt`.
+    /// Convenience method to map properties from `BloggingPromptRemoteObject`.
     ///
     /// - Parameters:
     ///   - remotePrompt: The remote prompt model to convert
     ///   - siteID: The ID of the site that the prompt is intended for
-    func configure(with remotePrompt: RemoteBloggingPrompt, for siteID: Int32) {
+    func configure(with remotePrompt: BloggingPromptRemoteObject, for siteID: Int32) {
         self.promptID = Int32(remotePrompt.promptID)
         self.siteID = siteID
         self.text = remotePrompt.text
-        self.title = remotePrompt.title
-        self.content = remotePrompt.content
+        self.title = String() // TODO: Remove
+        self.content = String() // TODO: Remove
```

Comment:
These will be removed in the next PR, as I remove the property from the Core Data model.